src/workflow/elements/gateways/LoopGateway.js

```diff
@@ -0,0 +1,22 @@
+const Joi = require('joi')
+const Gateway = require('./Gateway')
+
+class LoopGateway extends Gateway {
+
+  static get TYPE() {
+    return 'LOOP'
+  }
+
+  static get SCHEMA() {
+    return super.SCHEMA.concat(Joi.object({
+      type: Joi.string().allow(LoopGateway.TYPE).default(LoopGateway.TYPE),
+      maxIterations: Joi.number().integer().strict().default(1),
```

Comment:
maxIterations implies that there are breakout conditions within the loops, could just be iterations